entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/generator/provider/PropertyDefaultValueProvider.kt

```diff
@@ -0,0 +1,68 @@
+package godot.entrygenerator.generator.provider
+
+import org.jetbrains.kotlin.descriptors.PropertyDescriptor
+import org.jetbrains.kotlin.js.resolve.diagnostics.findPsi
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.referenceExpression
+import org.jetbrains.kotlin.resolve.BindingContext
+import org.jetbrains.kotlin.resolve.bindingContextUtil.getReferenceTargets
+import org.jetbrains.kotlin.resolve.descriptorUtil.fqNameSafe
+import org.jetbrains.kotlin.resolve.source.KotlinSourceElement
+import org.jetbrains.kotlin.serialization.deserialization.descriptors.DeserializedClassConstructorDescriptor
+
+object PropertyDefaultValueProvider {
```

Comment:
This is not required anymore (I left it after the refactoring proposal because we might needed it but after your changes in `RegistrationValuesHandler` they its not needed anymore)